GUIDE.md

```diff
@@ -1,50 +1,50 @@
+* * Request * - structure for transferring data from ViewController to Interactor
+* * Response * - structure for transferring data from Interactor to Presenter
+* * ViewModel * - structure for transferring data from Presenter to ViewController
+The Request, Response, ViewModel structures are declared inside the enum with the module name for the allocated namespace. DataFlow should clearly describe the business tasks and provide visual information about the data transmitted within the module.
 
 ## Interactor
-Содержит бизнес логику модуля. Нежелательно заставлять Interator ходить в сеть, обращаться к базе данных и т.д. Он выступает агрегатором бизнес логики. Так, например, для получения данных он обращается к Provider, к различным Worker'ам для выполнения каких-либо действий с данными, после чего передает подготовленные данные дальше в Presenter и т.д.
-Interactor получает запрос на операцию над данными от ViewController с помощью Request и предает данные в Presenter с помощью Response.
+It contains the business logic of the module. It is not advisable to force the Interator to go to the network, access the database, etc. He acts as an aggregator of business logic. So, for example, to get data, it calls the Provider, to various Workers to perform any actions with the data, and then sends the prepared data further to the Presenter, etc.
+Interactor receives a request for an operation on data from the ViewController with the Request and submits the data to the Presenter using Response.
 
 ## Worker
-Реализует переиспользуемую часть бизнес логики (например: алгоритм фильтрация данных).
+Implements the reused part of the business logic (for example: algorithm for data filtering).
 
 ## Presenter
-Содержит логику подготовки данных для отображения (например: форматирование данных (номера телефона, суммы), локализация и т.д.) и передает их во ViewController посредством ViewModel.
+It contains the logic for preparing data for display (for example: formatting data (phone number, sum), localization, etc.) and passes them to ViewController through ViewModel.
 
 ## ViewController
-Отвечает за отображение состояния, полученого от Presenter и взаимодействует с пользователем. ViewController содержит в себе только одну view, которая создается в методе loadView(). Он не управляет расположением view. Вся необходимая верстка уже должна быть в отдельно реализованной view. Однако ViewController выступает делегатом для всего view слоя. Так он знает, когда изменился текст в TextField или пользователь нажал на ячейку.
+It is responsible for displaying the status received from Presenter and interacting with the user. ViewController contains only one view, which is created in the loadView () method. It does not control the location of the view. All necessary layout should already be in a separately implemented view. However, ViewController acts as a delegate for the entire view layer. So he knows when the Text in TextField changed or the user clicked on the cell.
```

Comment:
`All necessary layout should be already presented ... So it knows when the Text in TextField is/was changed ...`